app/deepseg.cc

```diff
@@ -600,7 +609,11 @@ int main(int argc, char* argv[]) try {
 		}
 	}
 	// default green screen background (at capture true geometry)
-	cv::Mat bg = cv::Mat(capGeo.value().second, capGeo.value().first, CV_8UC3, cv::Scalar(0, 255, 0));
+	std::optional<std::pair<size_t, size_t>> bg_dim = capGeo;
+	if ( crop_region.height == 0) {
+		bg_dim = {crop_region.width, crop_region.height};
+	}
+	cv::Mat bg(bg_dim.value().second, bg_dim.value().first, CV_8UC3, cv::Scalar(0, 255, 0));
```

Comment:
```suggestion
	std::pair<size_t, size_t> bg_dim = *capGeo;
	if (!crop_region.height) {
		bg_dim = {crop_region.width, crop_region.height};
	}
	cv::Mat bg(bg_dim.second, bg_dim.first, CV_8UC3, cv::Scalar(0, 255, 0));
```

But actually using `cv::Size` for bg_dim would be semantically much more sane, than abusing `std::pair`.